### PR TITLE
bump maven to 3.5.2 to build successfully, 3.5.0 has vanished

### LINKED ISSUE #184: Bump maven to 3.5.2
It looks like `3.5.0` is gone [here](http://www-us.apache.org/dist/maven/maven-3/) and we need to use `3.5.2`.

### PR DESCRIPTION
**Pull request by awolfe76**
Closes #184

### CODE REVIEW

File: keycloak/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM jboss/keycloak-postgres:3.2.1.Final
+FROM jboss/keycloak-postgres:3.3.0.Final
```
Comment (hkeeler) [inline keycloak/Dockerfile]: Let's not bump the keycloak version up just yet.  There's likely more work to do there, and the `keycloak-postgres` image is [now deprecated](https://hub.docker.com/r/jboss/keycloak-postgres/) anyway.

**Comment by awolfe76** [comment]
@hkeeler - good catch. Didn't mean for that go up there. 😃 
Should be good now.

**Comment by hkeeler** [comment]
I'm surprised they take down old versions off the main download site.  Seems like they're just asking to break people stuff...  We should switch to https://archive.apache.org/dist/maven/maven-3/.

Per https://maven.apache.org/download.cgi:
>If you still want to use an old version you can find more information in the [Maven Releases History](https://maven.apache.org/docs/history.html) and can download files from the [archives](https://archive.apache.org/dist/maven/maven-3/) for versions 3.0.4+ ...

**Comment by hkeeler** [comment]
I'll merge as is so we can get the fix in.  I'll experiment with downloading Maven from there archive site as part of one my upcoming PRs.

**Pull request merged**